src/components/BlockingViews/BlockingView.tsx

```diff
@@ -10,51 +10,52 @@ import Navigation from '@libs/Navigation/Navigation';
 import useTheme from '@styles/themes/useTheme';
 import useThemeStyles from '@styles/useThemeStyles';
 import variables from '@styles/variables';
+import {TranslationPaths} from '@src/languages/types';
 
-const propTypes = {
+type BlockingViewProps = {
     /** Expensicon for the page */
-    icon: PropTypes.func.isRequired,
+    icon: React.FC<SvgProps>;
 
     /** Color for the icon (should be from theme) */
-    iconColor: PropTypes.string,
+    iconColor?: string;
 
     /** Title message below the icon */
-    title: PropTypes.string.isRequired,
+    title: string;
 
     /** Subtitle message below the title */
-    subtitle: PropTypes.string,
+    subtitle?: string;
 
     /** Link message below the subtitle */
-    linkKey: PropTypes.string,
+    linkKey?: TranslationPaths;
 
     /** Whether we should show a link to navigate elsewhere */
-    shouldShowLink: PropTypes.bool,
+    shouldShowLink?: boolean;
 
     /** The custom icon width */
-    iconWidth: PropTypes.number,
+    iconWidth?: number;
 
     /** The custom icon height */
-    iconHeight: PropTypes.number,
+    iconHeight?: number;
 
     /** Function to call when pressing the navigation link */
-    onLinkPress: PropTypes.func,
+    onLinkPress?: () => void;
 
     /** Whether we should embed the link with subtitle */
-    shouldEmbedLinkWithSubtitle: PropTypes.bool,
+    shouldEmbedLinkWithSubtitle?: boolean;
 };
 
-const defaultProps = {
-    iconColor: undefined,
-    subtitle: '',
-    shouldShowLink: false,
-    linkKey: 'notFound.goBackHome',
-    iconWidth: variables.iconSizeSuperLarge,
-    iconHeight: variables.iconSizeSuperLarge,
-    onLinkPress: () => Navigation.dismissModal(),
-    shouldEmbedLinkWithSubtitle: false,
-};
-
-function BlockingView(props) {
+function BlockingView({
+    icon,
+    iconColor,
+    title,
+    subtitle = '',
+    linkKey = 'notFound.goBackHome',
+    shouldShowLink = false,
+    iconWidth = variables.iconSizeSuperLarge,
+    iconHeight = variables.iconSizeSuperLarge,
+    onLinkPress = () => Navigation.dismissModal,
```

Comment:
Why have you removed the function call?
```suggestion
    onLinkPress = () => Navigation.dismissModal(),
```